bower.json

```diff
@@ -1,17 +1,18 @@
 {
-  "name": "preprint-service",
-  "dependencies": {
-    "ember": "~2.7.0",
-    "ember-cli-shims": "0.1.1",
-    "ember-qunit-notifications": "0.1.0",
-    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96",
-    "font-awesome": "~4.6.3",
-    "jquery": "^2.2.4",
-    "pretender": "~1.1.0",
-    "Faker": "~3.1.0",
-    "bootstrap-treeview": "^1.2.0",
-    "dropzone": "^4.3.0",
-    "bootstrap-sass": "^3.3.6",
-    "toastr": "^2.1.2"
-  }
+    "name": "preprint-service",
```

Comment:
@caneruguz The large differences here are because of formatting issues. When I initially added the jquery.tagsinput dependency, it was indented differently from everything else, which puzzled me. So I auto-indented the rest of the lines which is why it looks different.